lib/_http_common.js

```diff
@@ -94,6 +94,9 @@ function parserOnHeadersComplete(versionMajor, versionMinor, headers, method,
   incoming.httpVersionMajor = versionMajor;
   incoming.httpVersionMinor = versionMinor;
   incoming.httpVersion = `${versionMajor}.${versionMinor}`;
+  incoming.joinDuplicateHeaders = (socket && socket.server &&
```

Comment:
I suggest using optional chaining

Reply:
I used the same style as few lines above I can change both, probably when it was written optional chaining was not a thing

Reply:
I prefer optional chaining but yeah, this is just a suggestion